core/components/atoms/selectionCard/__tests__/SelectionCard.test.tsx

```diff
@@ -195,5 +234,11 @@ describe('selection card component hooks test', () => {
 
     fireEvent.click(element1);
     expect(element1).not.toHaveClass('Selection-card--selected');
+
+    const element3 = getByTestId('Selection-card-3');
+    fireEvent.click(element3);
+    expect(element3).not.toHaveClass('Selection-card--selected');
```

Comment:
Please check this test case once as on selection of element3 it should have `Selection-card--selected` class present. This test contradicts with the expected behavior. Please check how this test case is passing

Reply:
@anuradha9712  element3 is a disabled card so it must not be selected if it get clicked.